src/ploomber/spec/taskspec.py

```diff
@@ -200,11 +200,13 @@ def __init__(self,
                  meta,
                  project_root,
                  lazy_import=False,
-                 reload=False):
+                 reload=False,
+                 task_defaults=""):
```

Comment:
it'd be better to have `task_defaults=None` here

Reply:
Done!